api/openapi-spec/root_swagger.json

```diff
@@ -28176,6 +28176,39 @@
      }
     }
    },
+   "v1.ReplicationControllerCondition": {
+    "description": "ReplicationControllerCondition describes the state of a replication controller at a certain point.",
+    "required": [
```

Comment:
Sorry, that ship has sailed. The convention is "condition".
```
noun
1.
the state of something, especially with regard to its appearance, quality, or working order.
```
"State" is equally ambiguous.